teuthology/dispatcher/supervisor.py

```diff
@@ -21,39 +18,33 @@
 from teuthology.nuke import nuke
 
 log = logging.getLogger(__name__)
-start_time = datetime.utcnow()
-restart_file_path = '/tmp/teuthology-restart-workers'
-stop_file_path = '/tmp/teuthology-stop-workers'
 
 
 def main(args):
 
     verbose = args["--verbose"]
     archive_dir = args["--archive-dir"]
     teuth_bin_path = args["--bin-path"]
-    config_fd = int(args["--config-fd"])
+    config_file_path = args["--config-file"]
```

Comment:
Could you please rename the parameter to `--job-config` so we do not misinterpret it with teuthology config.

Reply:
Right, done!